md2notion/NotionPyRenderer.py

```diff
@@ -324,3 +331,40 @@ def render_image(self, token):
             'source': token.src,
             'caption': alt
         }
+
+    class __HTMLParser(HTMLParser):
+
+        def __init__(self):
+            super().__init__()
+            self.result = []
+
+        def handle_starttag(self, tag, attrs):
+            if tag != "img": 
+                return
+            src = next((value for key, value in attrs if key == "src"), "")
+            alt = next((value for key, value in attrs if key == "alt"), "No caption")
+            image = {
+                'type': ImageBlock,
+                'display_source': src,
+                'source': src,
+                'caption': alt
+            }
+            self.result.append(image)
+
+    def render_html(self, token):
+        content = token.content
+        parser = self.__HTMLParser()
+        parser.feed(content)
+
+        ret = parser.result
+        ret.insert(0, {
```

Comment:
So would this upload both the text HTML for the image as text and the ImageBlock rendered from the HTML too?

Ideally we would be able to remove the <img> tag from token.content after rendering it, so anyone uploading doesn't have to manually post-process their Notion notes after uploading.